### PR TITLE
Use Wedge elements for all MALI tests

### PR DESCRIPTION
**Pull request by mperego**
Albany is going to remove the support for Tet elements for MPAS generated meshes. 
In this PR we change all the land ice tests to use Wedge elements instead of Tet elements.

We also constrain the velocity on a patch of points, instead of on a single point, for the circular shelves test to remove the degree of freedom associated to rotations around the center of the shelf.

`Testing` perfomed on a linux machine linking the head of Albany master, as well as the branch of Albany where the Tets are disabled.

Checklist
* [x] User's Guide has been updated
* [ ] Developer's Guide has been updated
* [ ] API documentation in the Developer's Guide (`api.rst`) has any new or modified class, method and/or functions listed
* [x] Documentation has been [built locally](https://mpas-dev.github.io/compass/latest/developers_guide/building_docs.html) and changes look as expected
* [ ] The `E3SM-Project` submodule has been updated with relevant E3SM changes
* [ ] The `MALI-Dev` submodule has been updated with relevant MALI changes
* [x] Document (in a comment titled `Testing` in this PR) any testing that was used to verify the changes
* [ ] New tests have been added to a test suite

@matthewhoffman  Let me know if this change requires an update to the documentation.

### CODE REVIEW

**Comment by mperego** [comment]
@matthewhoffman we'd like to merge Albany changes soon. Is there anything holding up this PR?

**Comment by matthewhoffman** [comment]
### Testing

I tested the landice `full_integration` suite.
The expected tests fail baseline comparison but pass validation, as anticipated:
```
00:27 FAIL landice_dome_2000m_fo_decomposition_test
00:38 FAIL landice_dome_2000m_fo_restart_test
00:19 FAIL landice_dome_variable_resolution_fo_decomposition_test
00:17 FAIL landice_dome_variable_resolution_fo_restart_test
00:17 FAIL landice_circular_shelf_decomposition_test
01:02 FAIL landice_greenland_fo_decomposition_test
01:11 FAIL landice_greenland_fo_restart_test
```
I confirmed these changes can be blessed.

An update is required to `circular_shelf.cfg`.  I will push that.  

No changes to MALI-Dev or Albany are required, as this change makes use of existing functionality.

**Review by matthewhoffman — APPROVED** [review]
Approved based on review of changes and evaluation of full_integration suite results.  I'll merge after automated checks complete after my docs update.

**Pull request merged**
